funnel/views/membership.py

```diff
@@ -96,7 +96,7 @@ def new_member(self):
 
                     send_mail_async.queue(
                         sender=None,
-                        to=new_membership.user.email,
+                        to=new_membership.user.email.email,
```

Comment:
The convention we're using is `str(user.email)`. It's documented in the UserEmail model.